app/models/gebruiker.js

```diff
@@ -11,7 +11,7 @@ export default class Gebruiker extends Model {
   @hasMany('search-query') searchQueries;
 
   get group() {
-    return this.get('bestuurseenheden.firstObject');
+    return this.hasMany('bestuurseenheden').value().firstObject;
```

Comment:
This allows us to access the value of the relationship in a sync way (assuming the data is loaded) so we can safely use it in getters.